src/clusterfuzz/_internal/cron/triage.py

```diff
@@ -398,29 +411,40 @@ def main():
     # metadata works well.
     if not testcase.group_id and not dates.time_has_expired(
         testcase.timestamp, hours=data_types.MIN_ELAPSED_TIME_SINCE_REPORT):
+      logs.info(f'Skipping testcase {testcase_id}, pending grouping.')
       continue
 
     if not testcase.get_metadata('ran_grouper'):
       # Testcase should be considered by the grouper first before filing.
+      logs.info(f'Skipping testcase {testcase_id}, pending grouping.')
       continue
 
     # If this project does not have an associated issue tracker, we cannot
     # file this crash anywhere.
-    issue_tracker = issue_tracker_utils.get_issue_tracker_for_testcase(testcase)
+    try:
```

Comment:
This is the part that solves the incident, along deleting the config
@jonathanmetzman